pkg/azuredx/models/macro_test.go

```diff
@@ -83,6 +83,17 @@ func TestMacroData_Interpolate(t *testing.T) {
 			returnIs:  assert.Equal,
 			returnVal: fmt.Sprintf("CatCount >= %v and CatCount <= %v", fromString, toString),
 		},
+		{
+			name: "should parse $__timeFilter(value-with-hyphens)",
+			macroData: NewMacroData(&backend.TimeRange{
+				From: fromTime,
+				To:   toTime,
+			}, 0),
+			errorIs:   assert.NoError,
+			query:     "$__timeFilter(value-with-hyphens)",
+			returnIs:  assert.Equal,
+			returnVal: fmt.Sprintf("value-with-hyphens >= %v and value-with-hyphens <= %v", fromString, toString),
```

Comment:
just checking: will the value-with-hyphens be escaped in the resulting kusto query? i found out that with the dashed columns (or table names or variable names) kusto understand only if they are inside `[` and `'`, so like this ``['value-with-hyphens']``.

Reply:
Ah no I don't think so. Thanks for pointing that out. 

It's documented [here](https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/schema-entities/entity-names#identifier-quoting), right?

Reply:
@fridgepoet yes, i think you found the right doc